### PR TITLE
feat(email control): added email control component

### PR DESCRIPTION
**Pull request by 100stacks**
# Implement `<hx-email-control>`

### LGTM's
* [x] Dev LGTM
* [x] Design LGTM

### CODE REVIEW

File: docs/components/email/index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+<p>
+  The {{page.title}} component provides supplmental markup to define
+  and style an email address input control.
+</p>
+{% endblock %}
+
+{% block content %}
+<section>
+  <header>
+    <h2 id="basic-text-input">Basic Email Input</h2>
```
Comment (BY00565233) [inline docs/components/email/index.html]: Please change id
like this : id="basic-email-input" or id="basic-email"

File: docs/components/email/email-control-demo.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            hasHelpText: false,
+            hasErrorText: false,
+            isDisabled: false,
+            isRequired: false,
+            label: 'Email',
+            helpText: 'abc@abc.com',
+            errorText: 'Please enter valid email',
+            errorTextToDisplay: false,
+            helpTextToDisplay: false,
+            isHelpText : '',
```
[the remark this answers is not in the document]
Comment (100stacks) [reply docs/components/email/email-control-demo.js]: Removed `isHelpText`.

File: docs/components/email/index.html (hunk 1 above)
Comment (100stacks) [reply docs/components/email/index.html]: Changed to `id="basic-email-input"`.

**Comment by araiford** [comment]
Hey @100stacks 

The input itself looks great!  The only changes i have are content related.  Just two:

image

**Comment by 100stacks** [comment]
> Hey @100stacks
> 
> The input itself looks great! The only changes i have are content related. Just two:
> 
> image

@araiford, ok sounds good.  Changes made, please review...thanks!

**Pull request merged**